common/scala/src/main/scala/whisk/core/entity/AuthKey.scala

```diff
@@ -31,14 +27,14 @@ import spray.json.deserializationError
  * The constructor is private so that argument requirements are checked and normalized
  * before creating a new instance.
  *
- * @param (uuid, key) the uuid and key, assured to be non-null because both types are values
+ * @param k (uuid, key) the uuid and key, assured to be non-null because both types are values
  */
 protected[core] class AuthKey private (private val k: (UUID, Secret)) extends AnyVal {
-  def uuid = k._1
-  def key = k._2
+  def uuid: UUID = k._1
+  def key: Secret = k._2
   def revoke = new AuthKey(uuid, Secret())
-  def compact = s"$uuid:$key"
-  override def toString = uuid.toString
+  def compact: String = s"$uuid:$key"
```

Comment:
fwiw i tend to skip the signature where it's obvious (ie one liners, here .toString, or s"") but OK for uniformity.

Reply:
IntelliJ complains about no signature on public members, so I lazily fix files throughout.